src/mrpro/utils/_databuffermixin.py

```diff
@@ -0,0 +1,104 @@
+from collections import OrderedDict
+
+import torch
+
+from mrpro.data._Data import Data
+
+
+class DataBufferMixin(torch.nn.Module):
+    """A Mixin that allow to set a mrpro.data.Data as a Buffer.
```

Comment:
mrpro.Data.data

Reply:
Isnt the object called 'Data' in mrpro.data?
So you we can register mpro.data.Data as a Buffer.

(mrpro.data.Data.data would be a tensor, which we could register as a buffer without this code)

Or what am I missing?

Reply:
You are right, I forgot about one ".data"